### ISSUE TITLE
Option to select Vorbis quality

### ISSUE THREAD

**Issue by herrernst** [plietar/librespot#22]
By default, the lowest quality seems to be used, see generated `metadata.rs`:

``` rust
self.format.unwrap_or(AudioFile_Format::OGG_VORBIS_96)
```

Add possibility (command line switch?) to use 160 or 320.

**Comment by herrernst** [comment]
From the large size of the downloaded files in the tmp folder it seems that already 320 is used, but I didn't find a source reference after first glance.

**Comment by plietar** [comment]
Each track has a list of files, with various quality and format options.
https://github.com/plietar/librespot/blob/525b27/protocol/proto/metadata.proto#L79
https://github.com/plietar/librespot/blob/525b27/protocol/proto/metadata.proto#L147

The player currently always selects the first one, which happens to always be a 320kbps vorbis file.

``` rust
let file_id = track.files[0];
```

https://github.com/plietar/librespot/blob/525b27/src/player.rs#L118

But I agree, it should instead iterate over the list, making sure the file is in vorbis format (it can't decode anything else) and have a configuration option to select other qualities.

**Comment by plietar** [comment]
Implemented in f895616
